R4ML/R/sysml.bridge.R

```diff
@@ -82,10 +82,10 @@ sysml.MatrixCharacteristics <- setRefClass("sysml.MatrixCharacteristics",
 sysml.MLContext <- setRefClass("sysml.MLContext",
   fields = list(env="environment"),
   methods = list(
-    initialize = function(sparkContext = sysmlSparkContext) {
+    initialize = function(sparkContext = r4ml.env$sysmlSparkContext) {
       env <<- new.env()
       if (missing(sparkContext)) {
-        sparkContext = get("sysmlSparkContext", .GlobalEnv)
+        sparkContext <- r4ml.env$sysmlSparkContext
```

Comment:
since we have more system env we can keep it as it but that keep the user env in the new env i.e r4ml.user.env so that we can have read/write in the r4ml.user.env but only read for r4ml.env